pytest_docker_fixtures/__init__.py

```diff
@@ -72,3 +73,17 @@ def rabbitmq():
 def kafka():
     yield kafka_image.run()
     kafka_image.stop()
+
+
+@pytest.fixture(scope='session')
+def minio():
+    if IS_TRAVIS:
+        host = 'localhost'
+        port = 6379
```

Comment:
port `6379` for minio? Isn't that redis?

Does travis have minio support?

Reply:
AFAIK travis doesn't support Minio but we can run minio on `before_script` (maybe I'm wrong)

Reply:
> port 6379 for minio? Isn't that redis?
I forgot to change this line